### PR TITLE
add multiToken mediator

### PR DESCRIPTION
**Pull request by isentropy**

### CODE REVIEW

File: bridge/deploy_bridge_and_du2.sh [hunk 1]
```diff
@@ -7,31 +7,39 @@ cd `dirname $0`
...
 export ERC20_TOKEN_ADDRESS
 export HOME_ERC677_MEDIATOR=`jq -r .homeBridge.homeBridgeMediator.address < $ERC677RESULTS`
 export HOME_ERC677=`jq -r .homeBridge.bridgeableErc677.address < $ERC677RESULTS`
 export FOREIGN_ERC677_MEDIATOR=`jq -r .foreignBridge.foreignBridgeMediator.address < $ERC677RESULTS`
 
+echo "2.2 Multi Token"
+TASK=multiToken
+ENV="-e HOME_AMB_BRIDGE=$HOME_AMB_BRIDGE -e FOREIGN_AMB_BRIDGE=$FOREIGN_AMB_BRIDGE -e HOME_ERC677_TOKEN_IMAGE=$HOME_ERC677"
+docker run --name $TASK $ENV --env-file multiTokenMediator.env $CONTRACTS deploy.sh
+docker cp $TASK:/contracts/deploy/bridgeDeploymentResults.json $ERC677RESULTS
```
Comment (jtakalai) [inline bridge/deploy_bridge_and_du2.sh:41]: $ERC677RESULTS seems to be twice here. Will the results file be overwritten?

**Review by jtakalai — CHANGES_REQUESTED** [review]
Check the results file name

File: bridge/deploy_bridge_and_du2.sh [hunk 2]
```diff
@@ -7,31 +7,39 @@ cd `dirname $0`
...
-AMBTASK="deploy_amb"
 AMBRESULTS="bridgeDeploymentResultsAMB.json"
-
-ERC677TASK="deploy_erc677"
 ERC677RESULTS="bridgeDeploymentResultsERC677.json"
 
 echo "1. Deploying AMB"
-docker run --name $AMBTASK --env-file amb.env $CONTRACTS deploy.sh
-docker cp $AMBTASK:/contracts/deploy/bridgeDeploymentResults.json $AMBRESULTS
+TASK=amb
```
Comment (jtakalai) [inline bridge/deploy_bridge_and_du2.sh:14]: Task name changed. Is this a change in xdai images?

Comment (isentropy) [reply bridge/deploy_bridge_and_du2.sh:14]: it's just the name used to identify the docker process

**Review by jtakalai — APPROVED** [review]

**Pull request merged**